HISTORY.md

```diff
@@ -5,6 +5,7 @@
 * Listen on localhost by default to avoid firewall warnings ([#158](https://github.com/cucumber/cucumber-cpp/pull/158) Nik Reiman)
 * Better integrate Qt into buildsystem, it can now be disabled, and test it in CI ([#160](https://github.com/cucumber/cucumber-cpp/pull/160) Kamil Strzempowicz & Giel van Schijndel)
 * Support taking regex captures as arguments to the step definition's function ([#159](https://github.com/cucumber/cucumber-cpp/pull/159) Giel van Schijndel)
+* Track test coverage using coveralls service ([#162](https://github.com/cucumber/cucumber-cpp/pull/162) Kamil Strzempowicz)
```

Comment:
This is not a feature of `cucumber-cpp`. It's not (directly) useful for the users of this library. Instead it's useful for us, the developers, as such I believe this doesn't need to be mentioned in this file and definitely not in the "new features" section.